README.md

```diff
@@ -6,6 +6,9 @@ A module for converting cryptographic keys into human-readable phrases, similar
 to [Diceware](http://world.std.com/~reinhold/diceware.html) but with a larger
 wordlist. With Niceware, a 128-bit key is equivalent to an 8-word phrase.
 
+The wordlist contains 65,536 English words, which results in
```

Comment:
@grempe please take out this change, which will probably resolve the merge conflict, and then i can merge